contractcourt/channel_arbitrator.go

```diff
@@ -1767,8 +1767,8 @@ func (c *ChannelArbitrator) prepContractResolutions(
 					broadcastHeight: height,
 					htlcIndex:       htlc.HtlcIndex,
 					htlcAmt:         htlc.Amt,
-					ResolverKit:     resKit,
 				}
+				resolver.AttachResolverKit(resKit)
```

Comment:
hm, seems like this commit is reverted later, so might as well remove

Reply:
removed